.gitignore

```diff
@@ -8,6 +8,7 @@
 
 # intellij
 .idea/ant.xml
+/.idea/codestream.xml
```

Comment:
For consistency I guess this one should also be declared relative?
```suggestion
.idea/codestream.xml
```

Reply:
works for me. updated.